components/function-controller/pkg/utils/serving_utils.go

```diff
@@ -46,7 +46,7 @@ func GetServiceSpec(imageName string, fn serverlessv1alpha1.Function, rnInfo *Ru
 		Template: &servingv1alpha1.RevisionTemplateSpec{
 			Spec: servingv1alpha1.RevisionSpec{
 				RevisionSpec: v1beta1.RevisionSpec{
-					PodSpec: v1beta1.PodSpec{
+					PodSpec: corev1.PodSpec{
```

Comment:
is this really the same? v1beta1 was from githyb.com/knative/serving/apis/serving/v1beta1, now it is from kubernetes-core